### PR TITLE
TAD finalize pagination 

### PR DESCRIPTION
**Pull request by akenzua**
[TP11727](https://maps.tpondemand.com/restui/board.aspx?#page=board/5682182231901749200&appConfig=eyJhY2lkIjoiNzlGODk3MENDQkM0N0FCM0ZFQzkyNEVFODEyREM0M0EifQ==&boardPopup=userstory/11727/silent)

This PR implements the pagination feature on TAD consumer.

### CODE REVIEW

**Review by ladamsatmaps19 — APPROVED** [review]
Looks good to me! nice work

File: components/listingsPage/results.js [hunk 1]
```diff
@@ -26,20 +52,33 @@ const Results = ({ t }) => {
...
         }}
       >
-        {t("headings.showing")} 1 -10 {t("headings.of")} 1000{" "}
-        {t("headings.firms")}
+        <span>
+          {t("headings.showing")}
+          {firstIndex} - {lastIndex}
+          {t("headings.of")} {offered && offered.length} {t("headings.firms")}
+        </span>
+        <span>Firms presented in no particular order</span>
```
Comment (davidtrussler) [inline components/listingsPage/results.js:66]: Do we have the translation for this text? If not available yet can we ensure it's on a card to be added? 🙏

File: components/listingsPage/results.js [hunk 2]
```diff
@@ -8,14 +8,40 @@ import {
...
+    firstIndex = offered.length;
+  } else {
+    firstIndex = indexOfFirstFirm + 1;
+  }
+
+  const currentFirms =
+    offered && offered.slice(indexOfFirstFirm, indexOfLastFirm);
+
+  const totalPages = Math.ceil(offered && offered.length / firmsPerPage);
+  console.log(totalPages);
```
Comment (davidtrussler) [inline components/listingsPage/results.js:44]: Could we remove this before deploying? 🙂

File: components/listingsPage/results.js (hunk 1 above)
Comment (akenzua) [reply components/listingsPage/results.js:66]: Though this has been fixed, there is no translation for any text yet on TAD consumer. I guess that is the last thing they want me to do.

**Pull request merged**